sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -414,13 +414,30 @@ class StandardOptions(PipelineOptions):
 
   DEFAULT_RUNNER = 'DirectRunner'
 
+  ALL_KNOWN_RUNNERS = (
+      'apache_beam.runners.dataflow.dataflow_runner.DataflowRunner',
+      'apache_beam.runners.direct.direct_runner.BundleBasedDirectRunner',
+      'apache_beam.runners.direct.direct_runner.DirectRunner',
+      'apache_beam.runners.direct.direct_runner.SwitchingDirectRunner',
```

Comment:
I believe DirectRunner is aliased to SwitchingDirectRunner by default, should we only expose DirectRunner?

Reply:
I'll leave SwitchingDirectRunner in the list for now (since it was exposed before in the "Unexpected pipeline runner" error message). We can do additional documentation/cleanup in separate PRs.